Modules/_pickle.c

```diff
@@ -153,6 +153,9 @@ enum {
 
     /* Prefetch size when unpickling (disabled on unpeekable streams) */
     PREFETCH = 8192 * 16,
+    /* Data larger that this will be read in chunks, to prevent extreme
+       overallocation. */
+    SAFE_BUF_SIZE = 1 << 20,
```

Comment:
let's not use "SAFE" as name.  match whatever new name was similarly chosen in the Python code.